documentation/book/assembly-deploying-the-topic-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
+//:context: deployment
+include::common/attributes.adoc[]
+
+[id='deploying-the-topic-operator-{context}']
+= Deploying the Topic Operator
+
+:context: deploying
```

Comment:
Ignore what I said in my comment about the includes, this way works as well!!! Good call.